src/app/App.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import { addClasses, delegate, match, on, removeClasses } from 'metal-dom';
-import { array, async, isDefAndNotNull, isString } from 'metal';
+import { array, async, object, isDefAndNotNull, isString } from 'metal';
```

Comment:
Can you move this to the end of the list? I think we do sort this in general :)

Reply:
Of course! 😄